Tools/Scripts/webkitpy/xcode/simulated_device.py

```diff
@@ -420,30 +436,44 @@ def initialize_devices(cls, requests, host=None, name_base='Managed', simulator_
         if not hasattr(requests, '__iter__'):
             requests = [requests]
 
+        # Parse user-specified UDIDs
+        if type(udids) is str:
```

Comment:
To make things easier (avoid using commas and conditional splitting), you could specify `nargs='+'` in `optparse.make_option`.

Reply:
This is a good point! I was worried about messing up the positional argument flow. Turns out, this [may not even be possible](https://stackoverflow.com/a/1025232) with `optparse`...